src/main/java/org/jodreports/opendocument/OpenDocumentIO.java

```diff
@@ -106,7 +106,7 @@ private static void writeZipEntry(ZipOutputStream zipOutputStream, OpenDocumentA
 			zipOutputStream.putNextEntry(zipEntry);
 			IOUtils.copy(entryInputStream, zipOutputStream);
 		}
-		IOUtils.closeQuietly(entryInputStream);
+		entryInputStream.close();
```

Comment:
We might consider using a _try-with-resources_ statement instead? See https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html for details.